tests_integration/test_connector_flows.py

```diff
@@ -176,12 +152,12 @@ def test_connector_flows_image_change(run_kopf, random_connector_name):
         # Change image tag
         # kubectl patch tc/test-connector-image-local -p '{"spec": {"image": {"tag": "1.63.0"}}}' --type=merge
         kubectl_patch(f"tc/{connector_name}", {"spec": {"image": {"tag": "1.63.0"}}})
-        time.sleep(10)
-        pod = kubectl_get("pod", connector_name)
+        time.sleep(5)
```

Comment:
Ditto

Reply:
No, if you patch the get the pod right after it might still be running - give k8s time to apply

Reply:
we patch the `tc` object and then its reconciler needs to run and patch the pod and only then pod restarts